bpf/lib/common.h

```diff
@@ -276,7 +276,7 @@ enum {
 #define DROP_NO_SERVICE		-158
 #define DROP_POLICY_L4		-159
 #define DROP_NO_TUNNEL_ENDPOINT -160
-#define DROP_PROXYMAP_CREATE_FAILED	-161
+#define DROP_PROXYMAP_CREATE_FAILED_	-161 /* unused */
```

Comment:
You might consider adding a comment to `pkg/monitor/api/drop.go` stating that this code is unused.

Reply:
Was not aware of the `monitor/api`, will have alook!

Reply:
Added a note, also to other unused values. Added a trailing '_' to the unused names to discourage accidental reuse.